metricbeat/module/prometheus/_meta/fields.yml

```diff
@@ -1,12 +1,20 @@
 - key: prometheus
   title: "Prometheus"
   description: >
-    Stats collected from Prometheus.
+    Stats scraped from a Prometheus endpoint.
   short_config: false
   release: ga
   settings: ["ssl", "http"]
   fields:
-    - name: prometheus
-      type: group
+    # Order is important here, labels will match first, the rest are double
```

Comment:
What about placing labels in the base level to avoid possible issues with this? This is also the recommendation in ECS.

Reply:
I would prefer to keep it like this for now, main reasons are:
- It seems there is no way to do alias on `labels.*`
- If other modules push labels there, it could lead to mapping issues if they are using dots

I want to make sure this module won't end on mapping issues and we can use these metrics reliably, not against this change once the problem is gone (for instance, if/when index per module is implemented).

WDYT?

Reply:
LGTM, but I wonder some things (not blocking this PR)

>  It seems there is no way to do alias on labels.*

Why would we want to do it here?

>  If other modules push labels there, it could lead to mapping issues if they are using dots

I guess this is no different with any other module, maybe we shouldn't have a field for labels in ECS till we don't have a complete solution for these mapping issues.